services/app-web/src/constants/routes.ts

```diff
@@ -29,23 +30,25 @@ const STATE_SUBMISSION_FORM_ROUTES = [
     'SUBMISSIONS_CONTACTS',
     'SUBMISSIONS_DOCUMENTS',
     'SUBMISSIONS_REVIEW_SUBMIT',
+    'SUBMISSIONS_SUMMARY',
 ] as RouteT[]
 
 const RoutesRecord: Record<RouteT, string> = {
     ROOT: '/',
     AUTH: '/auth',
     DASHBOARD: '/dashboard',
     HELP: '/help',
-    SUBMISSIONS: '/submissions',
-    SUBMISSIONS_NEW: '/submissions/new',
-    SUBMISSIONS_FORM: '/submissions/:id',
-    SUBMISSIONS_TYPE: '/submissions/:id/type',
-    SUBMISSIONS_CONTRACT_DETAILS: '/submissions/:id/contract-details',
-    SUBMISSIONS_RATE_DETAILS: '/submissions/:id/rate-details',
-    SUBMISSIONS_CONTACTS: '/submissions/:id/contacts',
-    SUBMISSIONS_DOCUMENTS: '/submissions/:id/documents',
-    SUBMISSIONS_REVIEW_SUBMIT: '/submissions/:id/review-and-submit',
-    SUBMISSIONS_REVISION: '/submissions/:id/revisions/:revisionVersion',
+    SUBMISSIONS: 'submissions',
+    SUBMISSIONS_NEW: 'submissions/new',
+    SUBMISSIONS_FORM: 'submissions/:id/form/*',
+    SUBMISSIONS_TYPE: 'type',
```

Comment:
noticing that we've lost the ability to read these route constants as a holistic mapping of the routes in our application by making some of these constants partial routes. That may be needed for the v6 paradigm but I want to look more at this. Originally the intention of this record  was to be one place where you could see all the routes quickly.

Reply:
For visibility - I plan to look more into how we are using route constants on Monday. Want to do some more reading around the v6 paradigm as well.